projectq/ops/_qaagate.py

```diff
@@ -0,0 +1,31 @@
+#   Copyright 2019 ProjectQ-Framework (www.projectq.ch)
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from ._basics import BasicGate
+
+
+class QAA(BasicGate):
+    """
+    Quantum Aplitude Aplification gate.
+
```

Comment:
Could you add something along those lines to this gate's documentation string?

```
    .. warning::

      No qubit allocation/deallocation may take place during the call to :code:`algorithm`

```

Reply:
Yes, sure, that was my plan. In addition I will include the same in the amplitudeamplification.py file